frontend/controller/actions/identity.js

```diff
@@ -195,6 +195,13 @@ export default (sbp('sbp/selectors/register', {
         // we must re-sync our identity contract again to ensure we don't rejoin a group we
         // were just kicked out of
         await sbp('chelonia/contract/sync', identityContractID)
+        const rootGetters = sbp('state/vuex/getters')
+        const { mailbox } = rootGetters.currentIdentityState.attributes
+        if (mailbox && !contractIDs.includes(mailbox)) {
+          sbp('okTurtles.data/set', 'SYNCING_MAILBOX', true)
+          await sbp('chelonia/contract/sync', mailbox)
+          sbp('okTurtles.data/set', 'SYNCING_MAILBOX', false)
```

Comment:
Would be cool if we could DRY this somehow with Chelonia - since this seems to be a necessary trick that's used in multiple places.

For example, if you could check using Chelonia whether a specific `contractID` is currently being sync'd, would that allow us to remove these?